### PR TITLE
Fix interaction with groups that contain a slash

### LINKED ISSUE #11110: Unable to delete group with a slash in its name
### Steps to reproduce
1. Create a group with a slash in its name
2. Delete the group (example "mygroup/afterslash")
3. Refresh the page

### Expected behaviour
Group should be deleted.

### Actual behaviour
Group isn't deleted. It still appears after refreshing the page after a minute.

From the apache log when trying to delete the example group "mygroup/afterslash":
[07/Sep/2018:17:15:24 +0200] "DELETE /settings/users/groups/mygroup%2Fafterslash HTTP/1.1" 404 714 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"

Not found?

### Server configuration

**Operating system**: Debian Stretch

**Web server:** Apache 2.4.25

**Database:** MariaDB 10.1.26

**PHP version:** php 7.0.30

**Nextcloud version:** 13.0.6

**Updated from an older Nextcloud/ownCloud or fresh install:** fresh

It seems to be the same bug as here: https://github.com/nextcloud/server/issues/2954

### PR DESCRIPTION
**Pull request by nickvergessen**
@skjnldsv can you help? The Vue-router seems to escape the placeholders
So when you open a group which contains a slash and reload the page, it breaks.

https://localhost/index.php/settings/users/test%2Fusers
with 
https://localhost/index.php/settings/users/test/users it works quite fine.

Fixes #11110

### CODE REVIEW

File: settings/src/router.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default new Router({
 			name: 'users',
 			children: [
 				{
-					path: ':selectedGroup',
+					path: ':selectedGroup(.*)',
```
Comment (skjnldsv) [inline settings/src/router.js]: No, this is working fine.
We need to encode the variables in the ocs requests instead.
`Request URL: https://dev.skjnldsv.com/ocs/v2.php/cloud/groups/Test/1/users/details?offset=0&limit=25&search=`
Should be `/ocs/v2.php/cloud/groups/Test%2F1/users/details`

Comment (nickvergessen) [reply settings/src/router.js]: No, the old URL works since Nextcloud 9 and should not require urlencoding all of a sudden.

**Review by skjnldsv — APPROVED** [review]

**Comment by MorrisJobke** [comment]
It fixes #11110, right?

**Comment by skjnldsv** [comment]
@MorrisJobke no idea :/

**Comment by MorrisJobke** [comment]
@nickvergessen @skjnldsv Mind to rebase this one?

**Comment by MorrisJobke** [comment]
Rebased and rebuild the JS assets.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**